### DISCUSSION TITLE
How to enable print for hidden pages?

### DISCUSSION THREAD

**Discussion by cagix** [McShelby/hugo-theme-relearn#511] [Q&A] [answered]
I have some pages that should not be displayed in the menu because they are linked directly via the "More" section.

In order for them not to appear in the menu, even when I open these pages, I use the snippet
```
hidden: true
_build:
  render: always
  list: never
  publishResources: true
```
in the YAML header of these hidden pages (see also https://github.com/McShelby/hugo-theme-relearn/discussions/138).

When I click on the print icon in the open page, only an empty preview is displayed. How can I print these "hidden" pages nevertheless?

Edit: I've activated print support as shown in https://mcshelby.github.io/hugo-theme-relearn/basics/configuration/index.html#activate-print-support.

**Comment by McShelby** [comment, marked as answer]
I can confirm your issue but have to further investigate.

In the meanwhile it may help to store all "More" pages in a subfolder as it is done within the themes documentations exampleSite.

I've created issue #522 for this.
